### ISSUE TITLE
field.onDragStart throws error in Firefox input elements

### ISSUE THREAD

**Issue by robcmills** [erikras/redux-form#597]
**To reproduce**
In Firefox enter text into an redux-form input (with a field spread onto it), select some text and drag it.

It throws the following error: 

```js
Error: Permission denied to access property "setData"
```

**Comment by ooflorent** [comment]
Reproduced in `v5.2.3` and `v6.0.0-alpha.13`.
It may be a React bug.

**Comment by erikras** [comment]
Not sure there's anything this library can really do about that.

http://stackoverflow.com/questions/11927309/html5-dnd-datatransfer-setdata-or-getdata-not-working-in-every-browser-except-fi#11959389

**Comment by Elliot128** [comment]
Is there still no workaround for this? I saw there's an onDragStart handler but I can't find a way stop this error in Firefox.

**Comment by robcmills** [comment]
@Elliot128 

We just worked around it:
```javascript
const handleDragStart = (e) => {
	e.preventDefault()
	e.stopPropagation()
}
// jsx
<input
	...
	onDragStart={handleDragStart}
/>
```

**Comment by Elliot128** [comment]
Thanks @robcmills. This also worked for us without removing the ability to drag from fields.
```
const handleDragStart = (e) => {}
// jsx
<input
	...
	onDragStart={handleDragStart}
/>
```

**Comment by kylane** [comment]
This issue was closed, but it's still happening as of 7.3.0?
